src/mobile/src/main/java/tupperdate/android/home/Buttons.kt

```diff
@@ -0,0 +1,52 @@
+package tupperdate.android.home
+
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.Button
+import androidx.compose.material.ButtonConstants
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun BarButton(
+    size: Int,
+    onClick: () -> Unit,
+    backgroundColor: Color,
+    content: @Composable() () -> Unit,
```

Comment:
I'm not sure, but shouldn't the content be the last element to be able to use Kotlin's trailing lambda ?

Reply:
I don't know, but this is working fine